### ISSUE TITLE
Content

### ISSUE THREAD

**Issue by dzid26** [theQRL/theqrl.org#102]
I believe everyone's first question visiting QRL is: 
How is QRL different from other quantum resiliant crypto? 
Followed by: why XYZ blockchain wouldn't just upgrade their code to be QR. 

A comparison table to other blockchains wouldn't hurt somewhere on the second page. 

Also, there are bunch of Forbes, Google store, etc "random crap" and conferences links that, in my opinion, should be dumped at the bottom of the site. 

The developers links (API, etc) should be grouped under one big button and put next to other three (
Wallet, mining, node..)

**Comment by jackalyst** [comment]
> How is QRL different from other quantum resiliant crypto?
> Followed by: why XYZ blockchain wouldn't just upgrade their code to be QR.
>  A comparison table to other blockchains wouldn't hurt somewhere on the second page.

Thanks for the feedback. As a project, we've tended to focus on what QRL can do, vs what other blockchains can *not*. This means comparative aspects it out of scope for the main qrl website, especially in the form of a table. 

With that said, a longer form introduction to QRL (similar to what is currently in Discord) I think would be a boon to answer many of the commonly asked questions like "why  XYZ blockchain wouldn't just upgrade their code to be QR."

> Also, there are bunch of Forbes, Google store, etc "random crap" and conferences links that, in my opinion, should be dumped at the bottom of the site.

Forbes, etc, has multiple purposes. One is that articles written about QRL by third parties gives extra authority and credibility. The other is that it's a good way to link back to those who write about us. And finally, when a publication sees that we do this, it provides added incentive to write about us.

Conferences, I kind of agree especially as it's getting a bit out of date. It also creates a bit of weird flow where it would make more sense to have 'top open source contributors' directly underneath. Maybe conferences can be shifted to a different page entirely.

> The developers links (API, etc) should be grouped under one big button and put next to other three (
Wallet, mining, node..)

Just to make sure we're on the same page here, you mean to 'tile' it differently? 
![bottom](https://user-images.githubusercontent.com/232510/60988679-518cfd80-a301-11e9-832d-a0ae2c210d2d.png)

**Comment by dzid26** [comment]
All I am suggesting is a really a cleanup.  It seems like things were just added on top old stuff without trying to start fresh.


I think the main page should be really simple - even big Download button and maybe smaller for _web wallet_ would be enough.  
Then TEAM - this is where discord button, team memebers and github contributors could be. 


I quickly sketched what I mean. 
- Fewer things on the main page.
- 4 tiles - no sub-tiles (no address format, no qrlib etc) - it is the main page!
- Much fewer buttons that probably no one clicks. 
- Links to download and the team. 
- Token Migration - remove.
- Maybe only **upcoming** conferences to be on the **main** page. 
-Play store / AppStore are redundant and could be added in Download page for the wallet. 


![2019-07-10 16_57_48-QRL - The Quantum Resistant Ledger](https://user-images.githubusercontent.com/841061/61004454-10621100-a334-11e9-954a-7d3637af21a8.png)![2019-07-10 16_58_13-QRL - The Quantum Resistant Ledger](https://user-images.githubusercontent.com/841061/61004460-11933e00-a334-11e9-9d11-c82715063d6d.png)![2019-07-10 16_58_32-QRL - The Quantum Resistant Ledger](https://user-images.githubusercontent.com/841061/61004463-135d0180-a334-11e9-9ea3-0fb82642ff46.png)![2019-07-10 16_58_39-QRL - The Quantum Resistant Ledger](https://user-images.githubusercontent.com/841061/61004471-1657f200-a334-11e9-9db3-0d1ef7fd445e.png)![2019-07-10 16_58_44-QRL - The Quantum Resistant Ledger](https://user-images.githubusercontent.com/841061/61004477-1821b580-a334-11e9-9b2f-9e48a3604a84.png)

It would be interesting to see click rate on the Forbes and other logos.
Usually, I associate logos with useless things that everone brag about on. (Only after some time I realized there are actually links on most of them. )

**Comment by jplomas** [comment]
Stale issue closed with site relaunch
